packages/containers/sandbox/mockBackend/infiniteScroll.js

```diff
@@ -0,0 +1,11 @@
+const users = require('./mock/infinite-scroll/users.json');
+
+module.exports = function addRoutes(app) {
+	app.get('/api/infinite-scroll/users', (req, res) => {
+		const { startIndex, stopIndex } = req.query;
+
+		const items = users.slice(parseInt(startIndex, 10), parseInt(stopIndex, 10) + 1);
+
+		res.json({ items, totalUserCount: users.length });
+	});
+};
```

Comment:
This mock endpoint returns a slice of the users.json list according to provided parameters.